frontend/viewer/src/lib/components/field-editors/audio-input.svelte

```diff
@@ -19,7 +19,9 @@
 
     get duration() {
       this.#durationSub();
-      return this.audio.duration;
+      let duration = this.audio.duration;
+      if (duration === Infinity) duration = NaN;
```

Comment:
What code can handle `NaN` values but not `Infinity`? What problem is this line aimed at solving?

Reply:
The bits UI slider uses max to create an array of values, for nan it creates none. It fails to create an infinite array. Reported here: https://github.com/huntabyte/bits-ui/issues/1663 and I added a code comment linking the issue